packages/web/src/components/common/Modal.tsx

```diff
@@ -1,40 +1,50 @@
-import React, { useRef } from 'react';
-import { useOnClickOutside } from '@utils/hooks/useOnClickOutside';
 
+import React from 'react';
 interface Props {
   open: boolean;
   closeHandler: () => void;
   children?: React.ReactNode;
 }
 
 export const Modal: React.FC<Props> = ({ open, closeHandler, children }) => {
-  if (typeof window === 'undefined' || !open) return null;
 
-  const modalRef = useRef() as React.MutableRefObject<HTMLInputElement>;
-  useOnClickOutside(modalRef, () => closeHandler());
+  if (!open) return null;
+
+  const checkClickedOnOutside = (e: any) => {
+    const modal = document.getElementById("modal") as HTMLElement;
+    if (e.target.id == "modal") {
+
+      modal.classList.add('hide');
+      closeHandler();
+
+    }
+
+  }
```

Comment:
Why are you using a separate hook for this? Feel free to edit @utils/hooks/useOnClickOutside, if you'd like.